test/integration/job/job_test.go

```diff
@@ -1749,6 +1757,32 @@ func TestJobPodReplacementPolicy(t *testing.T) {
 			},
 			wantActive: int(podCount),
 		},
+		"feature flag true, delete pods, verify active and failed status and recreate once failed": {
```

Comment:
```suggestion
		"feature flag true, recreate failed pods, and verify active and failed counters": {
```
nit: This test does not delete pods. Also, we first await for recreation, then assert on the `active` and `failed`, so I suggest to reorder.

Reply:
resolved